src/mmapped_matrix.hpp

```diff
@@ -72,6 +60,28 @@ class MmappedMatrix {
     CheckStatus(close_status, "close");
   }
 
+  void Resize(Eigen::Index rows, Eigen::Index cols) {
+    // Resizes file so it's just right for our vector.
+    rows_ = rows;
+    cols_ = cols;
+    byte_count_ = rows * cols * sizeof(Scalar);
+    auto ftruncate_status = ftruncate(file_descriptor_, byte_count_);
+    if (ftruncate_status != 0) {
+      Failwith("MmappedMatrix could not resize the file at " + file_path_);
+    }
+    mmapped_memory_ = static_cast<Scalar *>(mmap(  //
```

Comment:
What happens to the previous mmapped memory when we reassign like this? Do we not have to deallocate it if it's already allocated?

Reply:
Changed to `mremap`.

Reply:
This is strange-- it still looks like mmap on the GitHub PR interface.

Reply:
Never mind! I see it below.